specification.yaml

```
@@ -295,6 +295,11 @@ paths:
         description: timing point of the race result
         schema:
           type: string
+      - name: idsOnly
+        in: query
+        description: possibility to not retrieve timing point details, only references
+        schema:
+          type: boolean
```

La til default og required som kan vere kjekt å vite:
```suggestion
        schema:
          type: boolean
          default: false
        required: false
```